src/cb_admin/components/QrBox.js

```diff
@@ -37,18 +35,21 @@ const QrBox = ({ qrCodeUrl, print, send, error, status }) => {
   const hasError = status === resendQrCodeState.ERROR;
 
   return (
-    <Container>
+    <Grid>
       <Img alt="QR code" src={qrCodeUrl} />
-      <Button onClick={print}>PRINT QR CODE</Button>
-      <Button onClick={send} disabled={hasSent}>
-        {
-          hasSent
-            ? 'QR CODE SENT'
-            : 'RESEND QR CODE'
-        }
-      </Button>
+      <Row between={'xs'}>
+        <Button onClick={print}>PRINT QR CODE</Button>
```

Comment:
Can we remove line 18:

```js
 margin: 0 2.5%;
```

This is what is making the buttons not quite in line with the rest of the page.